### DISCUSSION TITLE
Windows 端通过 Quicker 工具快速收集想法和灵感

### DISCUSSION THREAD

**Discussion by lightory** [flomoapp/3rd-party-tools#2] [分享创造]
Windows端，使用Quicker软件运行快捷命令快速发送选中的文本或者自行输入的文本。
支持配置全局快捷键，一键运行，极速添加。
在 https://getquicker.net/Sharedaction?code=65e9bd9e-ef03-42ba-a1f1-08d89ff18571&fromMyShare=True 中点击动作复制按钮，复制到自己的Quicker动作单中。

第一次运行，请选中动作右键，选择配置，输入API（必选）
1. 标签非必选项；
2. 选中文本是否剪辑来源地址和网页标题，默认选中添加，不需要可以勾选为不选中状态。

两种实现方式（动作可以配置全局快捷键）：
1. 选中文本时候直接添加到Flomo，支持默认多标签，动作右键配置即可，不需要的可以不配置。
2. 未选中文本的时候，输入框添加（支持选择多标签，可以动作右键配置）

**Comment by 9Hua** [comment]
厉害厉害,我自己写了一个才发现你这已经写好了

**Comment by hoxis** [comment]
![image](https://user-images.githubusercontent.com/5113809/113953135-b7e1df80-9849-11eb-8f95-ceff8c56fc2d.png)

采集的链接，会带星号（*），导致在flomo中点击访问时，会404
![image](https://user-images.githubusercontent.com/5113809/113953152-bf08ed80-9849-11eb-8859-0796c7392456.png)
